src/main/java/frc/robot/RobotContainer.java

```diff
@@ -145,43 +141,18 @@ private void configureButtonBindings() {
     new JoystickButton(m_operatorController, Button.kA.value)
         .onTrue(new InstantCommand(grabberSubsystem::toggle, grabberSubsystem));
 
+    // Deploys arm when left bumper is held and retracts when released
+    new JoystickButton(m_operatorController, Button.kLeftBumper.value)
```

Comment:
Is there a functional purpose to this?

Reply:
We will need more buttons for other positions but having the arm auto retract so it isn't extended while driving is useful.